crates/bevy_pbr/src/render/mesh_vertex_output.wgsl

```diff
@@ -0,0 +1,11 @@
+#define_import_path bevy_pbr::mesh_vertex_output
+
+@location(0) world_position: vec4<f32>,
```

Comment:
Is there a good reason why you haven't included the `@builtin()`s? I suppose so that this import can be reused both for vertex output which may use the builtin for position to output the clip position, and for fragment input which may use the builtin for position for screenspace coordinates?

Reply:
the builtin are not always present, not always the same, and not mandatory. the change in wgpu only checks for inputs with a `location`

Reply:
nop, that doesn't pass syntax validation if the user also want to specify builtins